lib/widgets/layouts/two_column_layout_web.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+class TwoColumnLayoutWeb extends StatelessWidget {
+
+  final Widget mainView;
+
+  final Widget sideView;
+
+  const TwoColumnLayoutWeb({
+    super.key,
+    required this.mainView,
+    required this.sideView,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.only(left: 0, right: 16.0, top: 16.0, bottom: 16.0),
```

Comment:
**Recommended**: Should use `EdgeInsetsDirectional` to support later `RTL` languages